pkg/k8s/apps/statefulsets.go

```diff
@@ -192,3 +192,13 @@ func (i *StatefulSetApp) PatchAnnotations(ctx context.Context, c kubernetes.Inte
 func (i *StatefulSetApp) Destroy(ctx context.Context, c kubernetes.Interface) error {
 	return statefulsets.Destroy(ctx, i.sfs.Name, i.sfs.Namespace, c)
 }
+
+// GetCloned Returns from Kubernetes the cloned statefulset
+func (i *StatefulSetApp) GetCloned(ctx context.Context, c kubernetes.Interface) (App, error) {
+	clonedName := fmt.Sprintf("%s-%s", i.sfs.Name, "okteto")
```

Comment:
If I understand correctly, I think we should use `model.DevCloneName` here to not duplicate the logic

Reply:
yes, we can reuse that